src/components/common/ConflictResolver.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Checkbox,
+  FormControlLabel,
+  makeStyles,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@material-ui/core';
+import Check from '@material-ui/icons/Check';
+import DoneAllIcon from '@material-ui/icons/DoneAll';
+import Error from '@material-ui/icons/Error';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import clsx from 'clsx';
+import React, { useEffect } from 'react';
+
+import {
+  ConflictResolutionStrategy,
+  QuestionComparisonStatus,
+} from 'generated/sdk';
+
+const useStyles = makeStyles((theme) => ({
+  heading: {
+    fontWeight: 'bold',
+  },
+  icon: {
+    marginRight: theme.spacing(2),
+  },
+  error: {
+    color: theme.palette.error.main,
+  },
+  resolved: {
+    color: '#ffbb00',
+  },
+  check: {
+    color: '#00ff00',
```

Comment:
Maybe here is also good to use something like success color from the palette but not sure this is just my suggestion.

Reply:
Sounds like a good idea